devnotes/scom_access.md

```diff
@@ -0,0 +1,92 @@
+# Access to SCOM registers
+
```

Comment:
You could briefly describe at the start of the document what is the goal here and why are we trying this.

Reply:
Added a description